custom_components/variable/config_flow.py

```
@@ -1158,6 +1196,8 @@
             }
         )
 
+        return SENSOR_OPTIONS_PAGE_1_SCHEMA
```

Ruff

Unnecessary assignment to `SENSOR_OPTIONS_PAGE_1_SCHEMA` before `return` statement

Remove unnecessary assignment

(RET504)